utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -0,0 +1,245 @@
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.UniqueValueRenderer;
+import com.esri.arcgisruntime.utilitynetworks.UtilityAssociation;
+import com.esri.arcgisruntime.utilitynetworks.UtilityAssociationType;
+import com.esri.arcgisruntime.utilitynetworks.UtilityNetwork;
+import com.esri.arcgisruntime.utilitynetworks.UtilityNetworkSource;
+
+public class DisplayUtilityAssociationsSample extends Application {
+  
+  //Feature server for the utility network
+  private static final String FeatureServerUrl = "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer";
+  private MapView mapView;
+  private GraphicsOverlay associationsOverlay;
+  private UtilityNetwork utilityNetwork;
+  
+  // max scale at which to create graphics for the associations
+  private int maxScale = 2000;
+  
+  /**
+   * Opens and runs application.
+   *
+   * @param args arguments passed to this application
+   */
+  public static void main(String[] args) {
+    
+    Application.launch(args);
+  }
+  
+  @Override
+  public void start(Stage stage) {
+    
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      
+      // set title, size, and add scene to stage
+      stage.setTitle("Display Utility Association Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+      
+      // create a map with the Topographic Vector Basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createTopographicVector());
+      
+      // create a viewpoint from envelope
+      Viewpoint viewPoint = new Viewpoint(41.8057655, -88.1489692, 23);
+      
+      // create the utility network
+      utilityNetwork = new UtilityNetwork(FeatureServerUrl);
+      
+      // Load utility network async and get all of the edges and junctions in the network.
+      utilityNetwork.loadAsync();
+      utilityNetwork.addDoneLoadingListener(() -> {
+        List<UtilityNetworkSource> networkSources = utilityNetwork.getDefinition().getNetworkSources();
+        
+        for (UtilityNetworkSource nSource : networkSources)
+          // Check an add all edges that are not subnet lines to the map.
+          if (nSource.getSourceType() == UtilityNetworkSource.Type.EDGE && nSource.getSourceUsageType() != UtilityNetworkSource.UsageType.SUBNET_LINE) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+          // Add all junctions to the map.
+          else if (nSource.getSourceType() == UtilityNetworkSource.Type.JUNCTION) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
```

Comment:
Both of these `if` blocks are doing the same thing, so we could in theory put them together.
However I think the way this is right now is better to convey the intentions of the code!